packages/devtools/src/utils.ts

```diff
@@ -23,3 +23,24 @@ export function getRulesBySlots(node: DebugResult, result: SlotInfo[] = []): Slo
     return [...acc, ...getRulesBySlots(child)];
   }, result);
 }
+
+export function filterSlots(slots: SlotInfo[], searchTerm: string) {
+  if (!searchTerm.length) {
+    return slots;
+  }
+
+  const result: SlotInfo[] = [];
+
+  slots.forEach(({ slot, rules }) => {
```

Comment:
nit: might be worth using a `map().filter(Boolean)` combination to be more 'pure function' programming